Source/WebKit/NetworkProcess/cache/NetworkCache.cpp

```diff
@@ -284,6 +284,12 @@ static StoreDecision makeStoreDecision(const WebCore::ResourceRequest& originalR
             return StoreDecision::NoDueToHTTPStatusCode;
     }
 
+    // Our caching does not take into account fragments properly.
```

Comment:
It does, in general, because we don't normally care about fragments. This is only an issue with redirects and the comment should be clearer.